test/api_test.ts

```diff
@@ -184,7 +184,7 @@ describe('API test', () => {
               },
             },
           }),
-        "Sync table defines an identityName that conflicts with its schema's identity.name",
+        "Sync table SomeSync's schema's identity has a name (ConflictingIdentity) which must match the identityName of the table (MyIdentityName).",
```

Comment:
Better, but "SomeSync's schema's" is a little hard to parse. Riffing:

> Identity name mismatch for sync table SomeSync. Either remove the schema's identity.name (ConflictingIdentity) or ensure it matches the table's identityName (MyIdentityName).

CC @alicia-codaio who may have wordsmithing suggestions.

Reply:
That's fantastic wording, done!